### ISSUE TITLE
TFuel reward is irrational 

### ISSUE THREAD

**Issue by qwertystars** [thetatoken/guardian-mainnet-guide#130]
I am making 5 TFuel by staking a 1000 theta in one day, is this normal?

**Comment by qwertystars** [comment]
Or maybe there is catch please help me understand this.

**Comment by Paulo23456** [comment]
> Or maybe there is catch please help me understand this.

Hello mate sorry about your issue what's your email I faced some issues and I got it fixed somehow

**Comment by Paulo23456** [comment]
> > Or maybe there is catch please help me understand this.
> 
> Hello mate sorry about your issue what's your email I faced some issues and I got it fixed somehow 

Hello mate I faced same issues

**Comment by alex-bochkov** [comment]
This is a scammer
![image](https://user-images.githubusercontent.com/13791336/125487444-84fd60e2-1e5a-4522-a0d4-224694717f34.png)

**Comment by Paulo23456** [comment]
> Or maybe there is catch please help me understand this.

When did you notice this

**Comment by Paulo23456** [comment]
> This is a scammer
> 
> ![image](https://user-images.githubusercontent.com/13791336/125487444-84fd60e2-1e5a-4522-a0d4-224694717f34.png)
> 
> 

What do you mean shut up, I faced same issues maybe you're the scammer

**Comment by alex-bochkov** [comment]
@qwertystars
> I am making 5 TFuel by staking a 1000 theta in one day, is this normal?

To be honest, that looks more than I'm getting or what other estimate.
For example, https://guardianmonitor.io/ estimates that 1K THETA stake should give you approx $0.33[1.03 TFUEL] per day.

**Comment by qwertystars** [comment]
Yes I was also confused about this, 3 days ago I received 12 Tfuel in one day and I noticed that on that night my PC did not shutdown. So, I was thinking maybe it is for uptime??

**Comment by qwertystars** [comment]
For clarification for the last two days it was shut down.

**Comment by Paulo23456** [comment]
> For clarification for the last two days it was shut down.

Let me have your email
